digest/glossary.js

```diff
@@ -0,0 +1,28 @@
+const utils = require('./utils')
+
+module.exports = () =>
+  utils.readdir('/glossary')
+    .then(convertGlossaryFileNamesToTerms)
+    .then(indexById)
+
+const convertGlossaryFileNamesToTerms = glossaryFileName =>
+  glossaryFileName
+    .map(fileName => ({
+      fileName,
+      id: fileName,
+      value: fileName.replace(/-/g, ' '),
+      path: `/glossary/${fileName}`,
+      markdownFilePath: `/glossary/${fileName}/README.md`,
+    }))
+
+const indexById = terms => {
```

Comment:
dont we have a helper for this?

Reply:
We're following the pattern we found in modules.js. It doesn't appear that we have a helper function this.